crates/bootstrap_client/Makefile.toml

```diff
@@ -0,0 +1,5 @@
+extend = "../../Makefile.toml"
+
+[tasks.test-integration]
+command = "cargo"
+args = ["test", "--test", "connect"]
```

Comment:
Is there any way to do this without having to enumerate the specific integration tests? I'm not sure I understand why the unit and integration tests are separate... it seems fine to just run them all at the same time.

Reply:
Not until cargo adds an option to do that. I separated them because that's how I tend to run them but I could just have `cargo make` run all of them and if I want to run them differently then I will run `cargo test --lib` myself?

Reply:
That would work for me as well. I usually only run all tests at the end.